src/test/java/dk/sdu/se_f22/contentmodule/stopwords/SemesterProjekt1.4_stopWordFilter/src/StopWords.java

```diff
@@ -0,0 +1,161 @@
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Scanner;
+
+public class StopWords implements IStopWords {
+    public static final String COLUMN_NAME ="name";
+    public static final String TABLE_Eng_WORDS ="words_to_removed";
+    public static final String WORDS_FROM_TOKEN_REMOVE="SELECT "+ COLUMN_NAME +" FROM "+ TABLE_Eng_WORDS +" WHERE "+
+            COLUMN_NAME +" = ?";
+    public static final String INSERT_DA_WORDS= "INSERT INTO "+ TABLE_Eng_WORDS +'('+COLUMN_NAME+")" +
+            " VALUES(?) ON CONFLICT DO NOTHING";
+    public static final String QUERY_DA_WORDS="SELECT * FROM " + TABLE_Eng_WORDS;
+    public static final String REMOVE_DA_WORDS="DELETE FROM " + TABLE_Eng_WORDS +" WHERE "+ COLUMN_NAME + " = ?";

```

Comment:
No reason to save these queries in variables (especially public static final), since they are only executed one place in the code, and if jut put directly in the code it makes it easier to see what query is actually being executed.